lib/fileutils.rb

```diff
@@ -1017,13 +1057,15 @@ def rm_r(list, force: nil, noop: nil, verbose: nil, secure: nil)
   end
   module_function :rm_r
 
+  # Equivalent to:
   #
-  # Equivalent to
+  #   FileUtils.rm_r(list, force: true, **kwargs)
   #
-  #   FileUtils.rm_r(list, force: true)
+  # See FileUtils.rm_r for keyword arguments +noop+ and +verbose+,
+  # and especially for keyword argument +secure+,
+  # which relates to security and vulnerability.
   #
-  # WARNING: This method causes local vulnerability.
```

Comment:
Again, this warning seems important, and should not be removed with a good justification.